external-crates/move/crates/move-analyzer/prettier-plugin/tests/run_spec.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+const assert = require('assert');
+const linediff = require('line-diff');
+const fs = require('fs');
+const path = require('path');
+const plugin = require('../');
+const prettier = require('prettier')
```

Comment:
FYI [jest](https://jestjs.io/docs/expect#tomatchsnapshotpropertymatchers-hint) is a convenient snapshot testing library that might be nice to have later (also does the diffing, etc.)

Not suggesting we add this now, just putting it on your radar :-)